### PR TITLE
Fix Readme.md Contribution Guide Link

### PR DESCRIPTION
**Pull request by PongDev**
## Why did you create this PR

- Fix Wrong Document Link in Readme.md Result in wrong result

## What did you do

- Change link from old repository (frontend) to https://github.com/thinc-org/cugetreg/wiki/Contribution-Guide

## Demo

[https://dev.cugetreg.com](https://dev.cugetreg.com)

## Checklist

- [ ] Deploy a demo
- [ ] Check browsers compatibility
- [ ] Wrote coverage tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,16 +47,16 @@ We saw this problem and built [CU Get Reg](https://cugetreg.com) to make this pl
 
 CU Get Reg is open source - we welcome all developers. If you want to be a part of CU Get Reg and improve Chulalongkorn University society, you can start here!
 
-- [Contribution Guides](#https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
-- [Set up your local environments](https://github.com/thinc-org/cugetreg-frontend/wiki/Contribution-Guides)
+- [Contribution Guides](https://github.com/thinc-org/cugetreg/wiki/Contribution-Guide)
```
Comment (bombnp) [inline README.md:50]:
```suggestion
- [Contribution Guide](https://github.com/thinc-org/cugetreg/wiki/Contribution-Guide)
```

**Review by bombnp — APPROVED** [review]
1 suggestion. Other than that, LGTM.

Ty kub

**Review by Leomotors — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -47,16 +47,16 @@ We saw this problem and built [CU Get Reg](https://cugetreg.com) to make this pl
...
+- [Set up your local environments](https://github.com/thinc-org/cugetreg/wiki/Contribution-Guide)
 - [Explore issues](https://github.com/thinc-org/cugetreg-frontend/issues)
```
Comment (Leomotors) [inline README.md:52]:
```suggestion
- [Explore issues](https://github.com/thinc-org/cugetreg/issues)
```

File: README.md (hunk 1 above)
Comment (Leomotors) [reply README.md:50]: +

**Pull request merged**